pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/SegmentMetadataImpl.java

```diff
@@ -504,4 +511,10 @@ public String getBitmapInvertedIndexFileName(String column, String segmentVersio
     return column + V1Constants.Indexes.BITMAP_INVERTED_INDEX_FILE_EXTENSION;
   }
 
+  @Override
+  public String getPinotHadoopJarVersion() {
```

Comment:
Use @Nullable annotation on return - it documents to users that this may be missing